### PR TITLE
[DC-1698] observation_source_value Values Not Nulled In R2021q2r1_deid

### PR DESCRIPTION
**Pull request by dcampbell-vumc**
* Add cleaning rule to `REGISTERED_TIER_DEID_CLEANING_CLASSES` list
* Updated affected datasets in cleaning rule module

### CODE REVIEW

**Review by lrwb-aou — CHANGES_REQUESTED** [review]
I think the affected_datasets should include the registered tier.  :)

**Comment by lrwb-aou** [comment]
@dcampbell-vumc Also, you need to change your branch name and remove `deid` from the name.  That causes problems in the integration tests, so they fast fail now if `deid` is in your branch name.

File: data_steward/cdr_cleaner/cleaning_rules/deid/string_fields_suppression.py [hunk 1]
```diff
@@ -112,7 +113,10 @@ def __init__(self, project_id, dataset_id, sandbox_dataset_id):
 
         super().__init__(issue_numbers=ISSUE_NUMBERS,
                          description=desc,
-                         affected_datasets=[cdr_consts.CONTROLLED_TIER_DEID],
+                         affected_datasets=[
+                             cdr_consts.CONTROLLED_TIER_DEID,
+                             cdr_consts.CONTROLLED_TIER_DEID
```
[the remark this answers is not in the document]
Comment (dcampbell-vumc) [reply data_steward/cdr_cleaner/cleaning_rules/deid/string_fields_suppression.py:118]: 😹😅😅

**Comment by dcampbell-vumc** [comment]
@lrwb-aou Created new branch and PR due to incorrect branch name

**Pull request closed without merging**